src/axom/slic/docs/sphinx/index.rst

```diff
@@ -3,7 +3,20 @@
 .. ##
 .. ## SPDX-License-Identifier: (BSD-3-Clause)
 
-Slic User Documentation
-=======================
+================
+Slic User Guide
+================
 
-Slic provides a lightweight logging facility.
+.. include:: sections/synopsis.rst
+
+.. toctree::
+   :maxdepth: 3
+   :hidden:
+
+   sections/getting_started.rst
+   sections/architecture.rst
+   sections/slic_macros.rst
+   sections/faq.rst
```

Comment:
Are you planning to add an `FAQ` in the near future?  

If not, I'd recommend removing this section and file until it has content.